static/app/components/events/interfaces/frame/stacktraceLink.tsx

```diff
@@ -124,11 +125,25 @@ function shouldshowCodecovFeatures(
 }
 
 interface CodecovLinkProps {
+  event: Event;
+  organization: Organization;
   codecovStatusCode?: CodecovStatusCode;
   codecovUrl?: string;
 }
 
-function CodecovLink({codecovUrl, codecovStatusCode}: CodecovLinkProps) {
+const onOpenCodecovLink = (organization: Organization, event: Event) => {
```

Comment:
lets declare this function inside of `CodecovLink` and not pass the stuff

Reply:
Good idea, thank you